drivers/adc/adc_mcux_lpadc.c

```diff
@@ -57,7 +59,108 @@ struct mcux_lpadc_data {
 	lpadc_conv_command_config_t cmd_config[CONFIG_LPADC_CHANNEL_COUNT];
 };
 
+static int mcux_lpadc_acquisition_time_setup(const struct device *dev, uint16_t acq_time,
+					     lpadc_conv_command_config_t *cmd)
+{
+	const struct mcux_lpadc_config *config = dev->config;
+	uint32_t adc_freq_hz = 0;
+	float adc_cycle_ns = 0;
+	/* The unit of elements in the sample time array is ADCK. */
+	uint32_t acquisition_time_value = ADC_ACQ_TIME_VALUE(acq_time);
+	uint8_t acquisition_time_unit = ADC_ACQ_TIME_UNIT(acq_time);
+	uint32_t sample_time_array[] = {3, 5, 7, 11, 19, 35, 67, 131};
+	uint8_t sample_time_array_size = sizeof(sample_time_array) / sizeof(*sample_time_array);
```

Comment:
You can probably remove this variable, and just use `ARRAY_SIZE(sample_time_array)` instead